psl-evaluation/src/main/java/org/linqs/psl/utils/evaluation/statistics/DiscretePredictionStatistics.java

```diff
@@ -47,7 +47,7 @@
 	private final Set<GroundAtom> correctAtoms;
 	
 	public DiscretePredictionStatistics(int tp, int fp, int tn, int fn,
-			double threshold, Map<GroundAtom, Double> errors, Set<GroundAtom> correctAtoms) {
+		double threshold, Map<GroundAtom, Double> errors, Set<GroundAtom> correctAtoms) {
```

Comment:
This actually should be indented twice because it is a continuation of the previous line.